opus/import/populate_obs_instrument_VGUVS_prof.py

```diff
@@ -0,0 +1,611 @@
+            el = - (min_ea - 90.) # negative
+        else:
+            el = max_ea - 90. # positive
+    else:
+        el = - (min_ea - 90.) # negative
+
+    return el
+
+# Opening angle to solar: the angle between the ring surface to the direction
+# where incoming photons from the source. Positive if source is at the north
+# side of the ring, negative if it's at the south side. If source is at the
+# north side, it's 90 - inc. If source is at the south side, then oa is
+# - (90 - inc).
+def populate_obs_ring_geometry_VGUVS_solar_ring_opening_angle1_PROF(**kwargs):
+    if _is_Voyager_at_north(**kwargs):
+        inc = populate_obs_ring_geometry_VGUVS_incidence1_PROF(**kwargs)
+        return inc - 90.
+    else:
+        inc = populate_obs_ring_geometry_VGUVS_incidence2_PROF(**kwargs)
+        return 90. - inc
+
+
+def populate_obs_ring_geometry_VGUVS_solar_ring_opening_angle2_PROF(**kwargs):
+    if _is_Voyager_at_north(**kwargs):
+        inc = populate_obs_ring_geometry_VGUVS_incidence2_PROF(**kwargs)
+        return inc - 90.
+    else:
+        inc = populate_obs_ring_geometry_VGUVS_incidence1_PROF(**kwargs)
+        return 90. - inc
+
+# Ring elevation to solar, same to opening angle except, it's positive if
+# source is at north side of Jupiter, Saturn, and Neptune, and south side of
+# Uranus. Negative if source is at south side of Jupiter, Saturn, and Neptune,
+# and north side of Uranus.
+def populate_obs_ring_geometry_VGUVS_solar_ring_elevation1_PROF(**kwargs):
+    if _is_Voyager_at_north(**kwargs):
+        target_name = populate_target_name_from_index(**kwargs)
+        if target_name =='U RINGS':
+            inc = populate_obs_ring_geometry_VGUVS_incidence2_PROF(**kwargs)
+            el = 90. - inc # positive
+        else:
+            inc = populate_obs_ring_geometry_VGUVS_incidence1_PROF(**kwargs)
+            el = inc - 90. # negative
+    else:
+        inc = populate_obs_ring_geometry_VGUVS_incidence2_PROF(**kwargs)
+        el = 90. - inc # positive
+
+    return el
+
+def populate_obs_ring_geometry_VGUVS_solar_ring_elevation2_PROF(**kwargs):
+    if _is_Voyager_at_north(**kwargs):
+        target_name = populate_target_name_from_index(**kwargs)
+        if target_name =='U RINGS':
+            inc = populate_obs_ring_geometry_VGUVS_incidence1_PROF(**kwargs)
+            el = 90. - inc # positive
+        else:
+            inc = populate_obs_ring_geometry_VGUVS_incidence2_PROF(**kwargs)
+            el = inc - 90. # negative
+    else:
+        inc = populate_obs_ring_geometry_VGUVS_incidence1_PROF(**kwargs)
+        el = 90. - inc # positive
+
+    return el
+
+def populate_obs_ring_geometry_VGUVS_ring_intercept_time1_PROF(**kwargs):
+    return populate_time1_from_index(column='RING_EVENT_START_TIME', **kwargs)
+
+def populate_obs_ring_geometry_VGUVS_ring_intercept_time2_PROF(**kwargs):
+    return populate_time1_from_index(column='RING_EVENT_STOP_TIME', **kwargs)
+
+def _is_Voyager_at_north(**kwargs):
```

Comment:
Move this above first use so it's easier to find. Also don't capitalize Voyager - we don't normally have capital letters in function names unless it's something special like an abbreviation.